TDlist.test.js

```diff
@@ -0,0 +1,42 @@
+import displayList from './__mocks__/displayList';
+import TDlist from './src/TDlist';
+
+test('Add li tag', () => {
+  document.body.innerHTML = '<div>'
+   + '  <ul id="list"></ul>'
+  + '</div>';
+  const list = new TDlist(false, 'new task');
+  list.addTask();
+  const ul = document.getElementById('list');
+  displayList(ul, 1);
+  const nbr = document.querySelectorAll('#list li');
+  expect(nbr.length).toBe(1);
+});
+
+test('test local storage', () => {
+  const list = new TDlist(false, 'another task');
+  expect(list.addTask()).toBe(2);
+});
```

Comment:
According to the project requirements:

![image](https://user-images.githubusercontent.com/57421823/161367360-af999dd4-bb47-4ee0-94db-b58da866993c.png)

Therefore:

- [ ] Please, kindly make sure all your tests are grouped by a description method.

That way, we will meet this particular requirement and move closer to approval 💪